pycbc/frame/losc.py

```diff
@@ -14,11 +14,11 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
```

Comment:
A note that there's a lot of mentions of LOSC in here (including in the file name) which need changing. But probably not here.

Reply:
True, I do not want to change function/file/class names here.